tests/integration/test_schema.py

```diff
@@ -1219,16 +1351,26 @@ async def test_schema(registry_async_client, trail):
     # Creating a new schema works after deleting the only available version
     res = await registry_async_client.post(
         "subjects/{}/versions".format(subject),
-        json={"schema": '{"type": "string"}'},
+        json={"schema": '{"type": "string", "unique": "%s"}' % unique_field_factory()}
     )
     assert res.status == 200
     res = await registry_async_client.get("subjects/{}/versions".format(subject))
     assert res.json() == [4]
 
-    # Check version number generation when deleting an entire subjcect
-    subject = new_random_name("subject")
+
+@pytest.mark.parametrize("trail", ["", "/"])
+async def test_schema_version_numbering(registry_async_client, trail):
+    """
+    Test updating the schema of a subject increases its version number.
+    Deletes the subjects and asserts that when recreated, has a greater version number.
+    """
+    subject = create_subject_name_factory(f"test_schema_version_numbering-{trail}")()
+    unique_field_factory = create_field_name_factory(trail)
```

Comment:
`trail` is not really useful here.

Reply:
The idea is to make the field names unique for the two parallel tests.